### DISCUSSION TITLE
Could the app analyze 200,000 pps?

### DISCUSSION THREAD

**Discussion by wthahaha** [GyulyVGC/sniffnet#373] [General]
Could the app analyze 200,000 pps?

**Comment by GyulyVGC** [comment]
It also depends on your hardware.
The best thing to do to verify it is to try: you can stress the network and see if there are dropped packets.
Using a UDP traffic generator I managed to sniff 45000 packets per second without losing packets.
I think this limit is higher, but the packet generator I used didn't go beyond.

**Comment by GyulyVGC** [comment]
Please, if you test Sniffnet and discover something new about its packets limit, let me know.

**Comment by GyulyVGC** [comment]
Today I tried testing Sniffnet with `iperf` on a different machine and I could observe that the current packets per second limit is higher than 100k:

![Screenshot 2024-01-09 at 12 26 24](https://github.com/GyulyVGC/sniffnet/assets/100347457/d1c49afe-a65d-433b-a16b-522c118fc8b3)

**Comment by GyulyVGC** [comment]
Closing this discussion in favour of issue #489
